src/PlaywightSharp.ProtocolTypesGenerator/Program.cs

```diff
@@ -8,7 +9,11 @@ private static async Task Main(string[] args)
         {
             (IProtocolTypesGenerator, RevisionInfo)[] generators =
             {
-                (new ChromiumProtocolTypesGenerator(), new RevisionInfo()),
+                (new ChromiumProtocolTypesGenerator(), new RevisionInfo
+                {
+                    ExecutablePath = Path.Combine(Directory.GetCurrentDirectory(), "..","..","..","..", @"PlaywrightSharp.Tests\bin\Debug\netcoreapp3.1\.local-chromium\Win64-733125\chrome-win\chrome.exe"),

```

Comment:
Can't we use Chromium's BrowserFetcher to get the path? this will change a lot.

Reply:
the `BrowserFetcher` class is located in `PlaywrightSharp.Chromium` we would have to depend on that project.
it's possible.
the only issue is running the first time since we won't have the protocol classes (webkit & firefox)

Reply:
We can make a temporal change when we need it, we would need it to be dynamic at the end of the day.